docs-2.0/nebula-studio/quick-start/st-ug-plan-schema.md

```diff
@@ -6,18 +6,19 @@ A graph schema for Nebula Graph must have these essential elements:
 
 - Tags (namely vertex types) and their properties.
 
-- Edge types and their properties
+- Edge types and their properties.
 
-In this article, the [Social Network: MOOC User Action Dataset](https://snap.stanford.edu/data/act-mooc.html "Click to go to Stanford Network Analysis Platform (SNAP) website") and 97 distinct course names are used to introduce how to design a schema.
+In this article, you can install the sample data set [basketballplayer](https://docs-cdn.nebula-graph.com.cn/dataset/dataset.zip)and it is used to introducing how to design a schema.
```

Comment:
Lacks a space after the link.

"and it is used to introducing how to design a schema"
==>
"and use it to explore a pre-designed schema"?